tfhe/src/high_level_api/integers/types/base.rs

```diff
@@ -932,25 +1055,25 @@ generic_integer_impl_operation!(Mul(mul) => mul_parallelized);
 generic_integer_impl_operation!(BitAnd(bitand) => bitand_parallelized);
 generic_integer_impl_operation!(BitOr(bitor) => bitor_parallelized);
 generic_integer_impl_operation!(BitXor(bitxor) => bitxor_parallelized);
-generic_integer_impl_operation!(Shl(shl) => left_shift_parallelized);
-generic_integer_impl_operation!(Shr(shr) => right_shift_parallelized);
-generic_integer_impl_operation!(RotateLeft(rotate_left) => rotate_left_parallelized);
-generic_integer_impl_operation!(RotateRight(rotate_right) => rotate_right_parallelized);
 generic_integer_impl_operation!(Div(div) => div_parallelized);
 generic_integer_impl_operation!(Rem(rem) => rem_parallelized);
-
+generic_integer_impl_shift_rotate!(Shl(shl) => left_shift_parallelized);
+generic_integer_impl_shift_rotate!(Shr(shr) => right_shift_parallelized);
+generic_integer_impl_shift_rotate!(RotateLeft(rotate_left) => rotate_left_parallelized);
+generic_integer_impl_shift_rotate!(RotateRight(rotate_right) => rotate_right_parallelized);
+//
```

Comment:
comment what for ?

Reply:
It was meant to have a comment like `assign operations` to space out macro calls